src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -56,6 +56,10 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == '' Or '$(TargetGroup)' == 'netstandard1.7'">
```

Comment:
>'$(TargetGroup)' == '' Or '$(TargetGroup)' == 'netstandard1.7' [](start = 24, length = 62)

looks like the default build here is netstandard1.7, so you don't really need the second part of your condition.